make/kube

```diff
@@ -40,10 +40,14 @@ rm -rf "${FISSILE_OUTPUT_DIR}"
 fissile build "${BUILD_TARGET}"
 
 if [ "${BUILD_TARGET}" = "helm" ]; then
+    if [[ $FISSILE_STEMCELL =~ ^.*fissile-stemcell-opensuse.*$ ]] ; then
+      chart_name_suffix="-opensuse"
+    fi
+
     cat > "${FISSILE_OUTPUT_DIR}/Chart.yaml" << EOF
 apiVersion: v1
 description: A Helm chart for SUSE Cloud Foundry
-name: scf
+name: cf${chart_name_suffix}
```

Comment:
Does this work for the SLE case? I'd think that the `nounset` option makes it fail if it's not set in line 44.

Reply:
Fixed both